kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/activity/ConversationActivity.java

```diff
@@ -973,6 +1026,10 @@ public void isAttachment(boolean attachment) {
         this.isAttachment = attachment;
     }
 
+    public void isImageVideo(boolean isImageVideo) {
```

Comment:
Check for naming convention, for setting property it will be setX where X is variable name

Reply:
done